Python/ds/stack_via_queue.py

```diff
@@ -0,0 +1,48 @@
+# Program to implement a stack using  
+# two queue
+from queue import Queue
```

Comment:
You have to use the only list for writing your code as clearly mentioned no inbuilt function